doc/doxygen/tutorial/tutorial.h.in

```diff
@@ -182,18 +182,23 @@
  *       <td> Higher order mappings. Do not
  *       solve equations, but rather compute the value of pi to high
  *       accuracy.
+ *       <br/> Keywords: MappingQ, FE_Nothing, ConvergenceTable, GridOut, FEFaceValues
  *       </td></tr>
  *
  *   <tr valign="top">
  *       <td>step-11</td>
  *       <td> Solving a Laplace problem with
  *       higher order mappings. Using mean value constraints and
  *       intermediate representations of sparsity patterns.
+ *       <br/> Keywords: AffineConstraints, DoFTools::extract_boundary_dofs(),
```

Comment:
```suggestion
 *       <br/> Keywords: AffineConstraints, DoFTools::extract_boundary_dofs,
```

Reply:
I think we should stick with the parentheses, like we do everywhere else.